shards/lang/src/shards.pest

```diff
@@ -21,16 +21,16 @@ None          =  { "none" | "null" }
 ConstValue    =  { None | Boolean | VarName | Number | String | Seq | Table | Enum }
 Shards        =  { "{" ~ Sequence? ~ "}" }
 Block         = _{ Func | EvalExpr | Expr | Enum | Shard | TakeTable | TakeSeq | ConstValue | Shards }
-Iden          = @{ (ASCII_ALPHA | "_" | "*") ~ (ASCII_ALPHANUMERIC | "_" | "-" | ".")* }
+Iden          = @{ (ASCII_ALPHA | "_" | "*") ~ (ASCII_ALPHANUMERIC | "_" | "-" | ".")* ~ !"::" }
```

Comment:
Can you just run a quick test on performance, time. Sometimes pegs fuck up

Reply:
```
/a/tst>./script.sh
 == Testing new == 

real    0m0.053s
user    0m0.000s
sys     0m0.000s
 == Testing old ==

real    0m0.049s
user    0m0.000s
sys     0m0.000s
/a/tst>./script.sh
 == Testing old == 

real    0m0.051s
user    0m0.000s
sys     0m0.000s
 == Testing new ==

real    0m0.054s
user    0m0.000s
sys     0m0.000s
/a/tst>./script.sh
 == Testing old == 

real    0m0.050s
user    0m0.000s
sys     0m0.000s
 == Testing new ==

real    0m0.053s
user    0m0.000s
sys     0m0.000s
/a/tst>./script.sh
 == Testing old == 

real    0m0.049s
user    0m0.000s
sys     0m0.000s
 == Testing new ==

real    0m0.051s
user    0m0.000s
sys     0m0.000s
/a/tst>
```

Reply:
Tested some ambal scripts, all with similar margins, the above one is `card-stats-library\ \(aMzdScjAE\).shs`

Reply:
Running `shards format ./in.shs >/dev/null`